client/gitSensor/GitSensorClient.go

```diff
@@ -118,6 +138,15 @@ type RefreshGitMaterialResponse struct {
 	LastFetchTime time.Time `json:"lastFetchTime"`
 }
 
+type LatestCommitMetadataRequest struct {
```

Comment:
shuld we not merge it with CommitMetadataRequest

Reply:
done